sdlf-utils/ingestion-examples/preprocessing/fargate-unzip/parameters-dev.json

```diff
@@ -0,0 +1,46 @@
+[
+    {
+        "ParameterKey": "pTaskName",
+        "ParameterValue": "ingestion"
+    },
+    {
+        "ParameterKey": "pEnv",

```

Comment:
Do we need to reference the `pEnv` or can we use the existing SSM parameter for it (`/SDLF/Misc/pEnv`)?
https://github.com/awslabs/aws-serverless-data-lake-framework/blob/b4945543f0a8f15f3516c68323822ba4fba00a3e/sdlf-cicd/template-cicd-child-foundations.yaml#L891

Reply:
Done.